src/main/resources/openshift/builder/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM jboss-eap-7/eap70-openshift:1.4
+FROM jboss-eap-7/eap70-openshift:latest
```

Comment:
Hi David!

You are absolutely right for containers in general. Here we point to the official Red Hat supported ImageStream for EAP 7.0. This is why I thought it was acceptable to do so. I am expecting to only have fully tested, hardened, supported, containerized versions of EAP 7.0 (EAP 7.1 is in a separate Stream).